tests/phpunit/src/TestBase.php

```diff
@@ -21,7 +21,8 @@
   public function __construct($name = NULL, array $data = array(), $data_name = '') {
 
     parent::__construct($name, $data, $data_name);
-    $this->projectDirectory = dirname(dirname(dirname(__DIR__)));
+    $directory = realpath(dirname(__FILE__) . '/../../../');
```

Comment:
I'm confused here - the tests themselves all now live in vendor/acquia/blt/phpunit/src. Running `phpunit tests/phpunit` from the blt-project directory does not find the tests that live in blt.